core-bundle/src/Controller/BackendController.php

```diff
@@ -183,8 +183,6 @@ public function alertsAction(): Response
      * It will determine the current provider URL based on the value, which is usually
      * read dynamically via JavaScript.
      *
-     * @throws BadRequestHttpException
-     *
```

Comment:
This should stay IMO. In the case where someone calls this method manually, they should handle the `BadRequestHttpException`.

Reply:
Changed my mind I think. Controller and Listener don’t need `@throws`...

Reply:
I don't think anyone will ever call the method manually, but I'm OK with keeping the annotation here. 👍